Code/Mantid/MantidPlot/src/CurvesDialog.cpp

```diff
@@ -462,15 +477,21 @@ void CurvesDialog::removeCurves()
   showCurveRange(boxShowRange->isChecked());
   d_graph->updatePlot();
 }
-
+//Enable Disable buttons function
 void CurvesDialog::enableAddBtn()
 {
   btnAdd->setEnabled (available->count()>0 && !available->selectedItems().isEmpty());
 }
 
+// enables or disables the button when appopriate number of graphs are in graph contents 
```

Comment:
Could you use the same style of comments found [here](https://github.com/mantidproject/mantid/blob/master/Code/Mantid/Framework/Algorithms/src/Rebin.cpp#L33-L39), this will be picked up by Doxygen when the documentation is generated.